pkg/sdk/testint/streamlits_integration_test.go

```diff
@@ -286,7 +286,7 @@ func TestInt_Streamlits(t *testing.T) {
 		require.Equal(t, e.Name, detail.Name)
 		require.Equal(t, e.UrlId, detail.UrlId)
 		require.Equal(t, mainFile, detail.MainFile)
-		require.Equal(t, stage.Location(), detail.RootLocation)
+		require.Equal(t, stage.ID().FullyQualifiedName(), sdk.NewSchemaObjectIdentifierFromFullyQualifiedName(detail.RootLocation[1:]).FullyQualifiedName())
```

Comment:
Just for info, why was the location check changed into ID and the root location has to be parsed from the 2nd character?

Reply:
Because of our recent favorite error:
```
    streamlits_integration_test.go:289: 
        	Error Trace:	/Users/asawicki/Projects/terraform-provider-snowflake/pkg/sdk/testint/streamlits_integration_test.go:289
        	Error:      	Not equal: 
        	            	expected: "@\"int_test_db_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3\".\"int_test_sc_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3\".\"LSGTGWIT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3\""
        	            	actual  : "@\"int_test_db_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3\".\"int_test_sc_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3\".LSGTGWIT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-@"int_test_db_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3"."int_test_sc_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3"."LSGTGWIT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3"
        	            	+@"int_test_db_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3"."int_test_sc_IT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3".LSGTGWIT_BDACED40_D12A_DB50_2C54_5BD2D1188EA3
```
It's easier to workaround this with identifiers than with location (to be addressed with identifiers redesign with all such problems).